### PR TITLE
Force git hooks to contain unix styled commands

### PR DESCRIPTION
**Pull request by veewee**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Documented?   | no
| Fixed tickets | #458, #462, #447

This bug was introduced in version 0.13 : before we used to force the git hook command to be unix styled. 

(reference: https://github.com/phpro/grumphp/commit/415fd1a9c70f5905326f0651ac5009a70d42441e#diff-aed6b638f2d7de8614c3516d7939c1a0L135 )

### CODE REVIEW

File: spec/Process/ProcessBuilderSpec.php [hunk 1]
```diff
@@ -27,9 +27,19 @@ function it_is_initializable()
 
     function it_should_be_able_to_create_process_arguments_based_on_taskname(ExternalCommand $externalCommandLocator)
     {
-        $externalCommandLocator->locate('grumphp')->willReturn('/usr/bin/grumphp');
+        $externalCommandLocator->locate('grumphp', false)->willReturn('/usr/bin/grumphp');
 
-        $arguments = $this->createArgumentsForCommand('grumphp');
+        $arguments = $this->createArgumentsForCommand('grumphp', false);
+        $arguments->shouldHaveType(ProcessArgumentsCollection::class);
+        $arguments[0]->shouldBe('/usr/bin/grumphp');
```
Comment (Landerstraeten) [inline spec/Process/ProcessBuilderSpec.php]: `$arguments->shouldHaveKeyWithValue(0, '/usr/bin/grumphp');`

File: src/Process/ProcessBuilder.php [hunk 2]
```diff
@@ -43,11 +43,12 @@ public function __construct(GrumPHP $config, ExternalCommand $externalCommandLoc
     /**
      * @param string $command
      *
+     * @param bool $forceUnix
```
Comment (Landerstraeten) [inline src/Process/ProcessBuilder.php]: I think this should be on line 45 (swap the newline)

File: spec/Process/ProcessBuilderSpec.php (hunk 1 above)
Comment (veewee) [reply spec/Process/ProcessBuilderSpec.php]: To me, it's more readable as it is right now

Comment (Landerstraeten) [reply spec/Process/ProcessBuilderSpec.php]: 👍

**Pull request merged**